src/lib/features/chain_of_trust.js

```diff
@@ -113,6 +122,29 @@ class ChainOfTrust {
       debug(err);
       throw err;
     }
+
+    try {
+      await uploadToS3(task.queue, task.status.taskId, task.runId,
+        cotBufferStream, 'public/chain-of-trust.json', expiration, {
+          'content-type': 'text/plain',
+          'content-length': chainOfTrust.length
+        });
+    } catch (err) {
+      debug(err);
+      throw err;
+    }
+
+    try {
+      await uploadToS3(task.queue, task.status.taskId, task.runId,
+        sigBufferStream, 'public/chain-of-trust.json.sig', expiration, {
+          'content-type': 'application/octet-stream',
+          'content-length': chainOfTrustSig.length
+        });
+    } catch (err) {
+      debug(err);
+      throw err;
+    }
```

Comment:
the `try`/`catch` pairs aren't necessary here, the errors will be reported to either Papertrail or the task log.